readthedocs/projects/models.py

```diff
@@ -187,21 +187,25 @@ class AddonsConfig(TimeStampedModel):
     # Flyout
     flyout_enabled = models.BooleanField(default=True)
     flyout_sorting = models.CharField(
+        verbose_name=_("Sorting of versions"),
         choices=ADDONS_FLYOUT_SORTING_CHOICES,
-        default=ADDONS_FLYOUT_SORTING_ALPHABETICALLY,
+        default=ADDONS_FLYOUT_SORTING_SEMVER_READTHEDOCS_COMPATIBLE,
```

Comment:
This won't update existing projects, right? I wonder if we should migrate it for them 🤔

Reply:
Yeah, not sure if we have a good way to tell apart users that set this option explicitly and who didn't (maybe check if created and modified are equal?)